tests/acceptance/features/bootstrap/GuestsContext.php

```diff
@@ -657,4 +657,67 @@ public function userHasSharedFolderWithGuestUser(
 			"User $sharer failed to share $filePath with user $guestUser"
 		);
 	}
+
+	/**
+	 * @Given the administrator has set useWhitelist to :state
+	 *
+	 * @param string $state
+	 *
+	 * @return void
+	 */
+	public function theAdministratorHasSetUsewhitelistTo(string $state) : void {
+		$fullUrl = $this->featureContext->getBaseUrl() . '/index.php/apps/guests/config';
+		$body = [
+			'group' => 'guest_app',
+			'useWhitelist' => $state
+		];
+		$headers = [];
+		$headers['Content-Type'] = 'application/x-www-form-urlencoded';
+		$response = HttpRequestHelper::sendRequest(
```

Comment:
can't we use `put` method 
```suggestion
		$response = HttpRequestHelper::put(
```